packages/react/src/types.ts

```diff
@@ -109,35 +109,18 @@ export type ClerkProp =
   | undefined
   | null;
 
-type ButtonProps = {
-  mode?: 'redirect' | 'modal';
+export type SignInButtonProps = _SignInButtonProps & {
+  children?: React.ReactNode;
+};
+
+export type SignUpButtonProps = _SignUpButtonProps & {
   children?: React.ReactNode;
 };
```

Comment:
🙃
```suggestion
export type SignInButtonProps = React.PropsWithChildren<_SignInButtonProps>

export type SignUpButtonProps = React.PropsWithChildren<_SignInButtonProps>
```

IMO we can also remove these from this file and directly do `WithClerkProp<React.PropsWithChildren<SignInButtonProps>>` in the respective button tsx files